scripts/zones/Chamber_of_Oracles/mobs/Radiant_Wyvern.lua

```diff
@@ -0,0 +1,22 @@
+-----------------------------------
+-- Area: Chamber of Oracles
+--  Mob: Radiant Wyvern
+-- KSNM: Eye of the Storm
+-----------------------------------
+require("scripts/globals/status")
+require("scripts/globals/monstertpmoves")
```

Comment:
monstertpmoves isnt needed as a require in all the mob scripts

Reply:
they dont have a special ability before death, wiki is just referring to their unique breath attacks. will look at other alterations - wyverns already have boosted movement speed of 50 not sure if they need more but will compare against vid.